src/components/ConfigValidation/ConfigIndicator.tsx

```diff
@@ -151,15 +158,7 @@ export class ConfigIndicator extends React.PureComponent<Props, {}> {
           trigger={['hover', 'focus']}
           rootClose={false}
         >
-          <span style={{ color: this.getValid().color }}>
-            <Icon
-              type="pf"
-              name={this.getValid().icon}
-              style={{ fontSize: this.size() }}
-              className="health-icon"
-              tabIndex="0"
-            />
-          </span>
+          <span style={{ color: this.getValid().color }}>{createIcon(this.getValid(), 'sm')}</span>
```

Comment:
Same for the previous command, this should maybe be named `getValidationResult` or something like that.